i18n.md

```
@@ -215,14 +217,15 @@ catalog has a missing key or an untranslated (empty/`fuzzy`) entry.
 ```python
 from common.authentication import I18nJWTAuth
 
+
 @route.get("/endpoint", auth=I18nJWTAuth())
-def my_endpoint(request):
-    ...
+def my_endpoint(request): ...
 ```
 
 ✅ **Use `gettext_lazy` for all user-facing strings in Python code:**
 ```python
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)